### ISSUE TITLE
AutoPlot generation for model documentation does not include background (Trac #1128)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#1169]
Somewhere along the line in transitioning from backgrounds explicitly being defined in models and being managed by the sasmodels infrastructure the auto generation of the plot for documentation has lost that parameter.  The result is that now the plot does not match the default parameters in the table of the same docs.

Migrated from http://trac.sasview.org/ticket/1128
```json
{
    "status": "closed",
    "changetime": "2018-08-19T18:10:29",
    "_ts": "2018-08-19 18:10:29.948948+00:00",
    "description": "Somewhere along the line in transitioning from backgrounds explicitly being defined in models and being managed by the sasmodels infrastructure the auto generation of the plot for documentation has lost that parameter.  The result is that now the plot does not match the default parameters in the table of the same docs.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2018-07-04T19:17:02",
    "component": "SasView",
    "summary": "AutoPlot generation for model documentation does not include background",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "pkienzle",
    "type": "defect"
}
```

**Comment by RichardHeenan** [comment]
Trac update at `2018/08/06 15:56:33`: **richardh** commented:

This is of course only seen if a new plot has to be generated following changes to a model, hence we found it whilst sorting #1157 for the Guinier model.

**Comment by butlerpd** [comment]
Trac update at `2018/08/07 13:54:52`: 
* **butler** changed owner from "" to "pkienzle"
* **butler** changed status from "new" to "assigned"

**Comment by pkienzle** [comment]
Trac update at `2018/08/17 15:53:29`: **pkienzle** commented:

Fixed in sasmodels.  Close ticket when verified.

**Comment by butlerpd** [comment]
Trac update at `2018/08/19 18:10:29`: 
* **butler** commented:

Fixed by commit !7b9e4ddca7ad3551d8fe3696d2afe75b0018599e.  Tested that it does as advertized.  Was pushed directly to master rather than through a PR as was a rather simple fix and sasmodels cannot yet have builds for testing.  Basically created a DEFAULT_BACKGROUND and set to 1e-3 in direct_model.py rather than set it to 1e-3 in the two places it currently is set: direct_model.py and modelinfo.py)

Now closing ticket


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "assigned" to "closed"
